rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -69,16 +69,38 @@ namespace rocal{
         return ptr;
     }
 
+    template<typename T>
+    void copy_data_numpy_wrapper(rocalTensor& output_tensor, T array) {
+        auto buf = array.request();
+        T *ptr = (T *)buf.ptr;
+        output_tensor.copy_data((void *)ptr, RocalOutputMemType::ROCAL_MEMCPY_HOST);
+    }
+
+    void copy_data_cupy_wrapper_u8(rocalTensor& output_tensor, long array) {
```

Comment:
Why can we not use templates in copy_data_cupy_wrapper too similar to numpy.

Reply:
Is there a particular reason we have introduced seperate API calls for this?

Reply:
Cupy arrays are representable as py::array<T> arrays like numpy arrays